clayui.com/src/components/clay/Editor.js

```diff
@@ -109,10 +114,27 @@ const Editor = ({code, disabled = false, imports, preview = true, scope}) => {
 					<LiveError />
 				</div>
 			)}
+
 			<div className="gatsby-highlight">
 				<div style={{padding: '10px'}}>
-					{imports && <LiveEditor disabled value={imports} />}
-					<LiveEditor />
+					<ClayButtonWithIcon
+						className="btn-collapse"
+						displayType="unstyled"
+						onClick={() =>
+							setCollapseCode(collapseCode => !collapseCode)
+						}
+						small
+						spritemap={spritemap}
+						symbol={collapseCode ? 'angle-right' : 'angle-down'}
+						title={collapseCode ? 'Expand' : 'Collapse'}
+					/>
+
+					{!collapseCode && (
+						<>
+							{imports && <LiveEditor disabled value={imports} />}
+							<LiveEditor />
```

Comment:
Typically we add new lines between sibling elements or components, so we would need one above this line.

Reply:
That's how it was before I started working on it so I left it like that. But yea, I agree, empty lines between siblings is the way to go :)

Reply:
Its typically not strictly enforced and likely a handful of instances of gone into upstream, but if I notice it I'll likely recommend it. But overall, not a huge deal, its more of a portal SF rule that we try to also adopt here.